decision/12.md

```diff
@@ -0,0 +1,55 @@
+---
+published: true
+layout: default-theme-wet-boew-en
+title: Design decision 12 - Double H1 on a single page
+description: Confirming the possibility to use two H1s on a single page and conform to accessibility requirements
+modified: 2020-06-30
+---
+
+## Summary
+
+* **Author:** @GormFrank
+* **Expert:** @duboisp, @bsouster, @namjohn920, @GormFrank
+* **Status:** Draft
+* **Last updated:** 2020-06-30
+* **History:**
+	* (2020-06-30) Initial draft
+
+# History
+
+There was a worry about the use of two heading level 1 on a single page, one being for the navigation in the section inside which the page sits (service initiation template) and one describing the content of the page. The HTML5 specification supports that pattern, but the concern was that browsers still don’t support it or badly support it. We need to keep in mind that we still need to support IE11. Last summer, IE11 still represented more than 15% of all the traffic on Canada.ca
+
+The proper way of doing it is by using H1s in combination of sectioning element as illustrated in the whatwg spec (see reference below).
+
+## Rational
+
+It was confirmed through research, discussions and accessiblity assessments that having two H1 tags on a single page is acceptable. By acceptable, it is meant that rightfully using double H1s on a page:
+* will not make you fail WCAG2.1 AA, as per an accessibility assessment led by the Accessibility, Accommodations & Adaptative Computer Technology (AAACT) Program team on February 26, 2020;
+* has the potential to improve your Search Engine Optimization (SEO) performances, as per a research led by the Principal Publisher team updated on February 18, 2020.
+
+It was presented by CRA team on January 9, 2020 and Principal Publisher has formally documented the evidence, February 18, 2020, to support the hypothesis.
+
+## Guidance
+
+These scenarios **do not** represent WCAG failures:
+
+* A lack of headings (unless you’re striving for Level AAA)
+* Missing "h1"
+* More than one "h1"
+* Skipped heading levels
+* Multiple headings with the same text
+
+These scenarios **do** represent WCAG failures:
+
+* Heading-like text that isn’t marked up as headings
+* Illogical heading order
+* Nondescriptive headings
+
```

Comment:
Add an example and explain the advantage.

Like with the subway navigation where their h1 represent a set of pages and another h1 for the main content. Google Can show the h1 with the set of pages followed by the main content h1 heading or showing the main content h1 heading as a section and under it the h1's used for subway navigation.
For example
* Service name A (Apply, Eligibility...)
* Service name B (Apply, Eligibility...)
* Apply (Service name A, Service name B, Service name C...)
* Eligibility (Service name A, Service name B, Service name C...)